src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Controllers/AggregatedEmployerRequestController.cs

```diff
@@ -0,0 +1,38 @@
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Provider.Shared.UI;
+using SFA.DAS.Provider.Shared.UI.Attributes;
+using SFA.DAS.ProviderRequestApprenticeTraining.Application.Queries.GetEmployerRequests;
+using SFA.DAS.ProviderRequestApprenticeTraining.Web.Authorization;
+using SFA.DAS.ProviderRequestApprenticeTraining.Web.Infrastructure;
+using SFA.DAS.ProviderRequestApprenticeTraining.Web.Models;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace SFA.DAS.ProviderRequestApprenticeTraining.Web.Controllers
+{
+    [Authorize(Policy = nameof(PolicyNames.HasProviderAccount))]
+    [SetNavigationSection(NavigationSection.EmployerDemand)]
```

Comment:
There will not be a specific navigation menu for RAT - so the navigation section needs to be for home - what ever that is

Reply:
This has been fixed